README.md

```diff
@@ -49,23 +49,37 @@ You can build your precompile and Subnet-EVM with `./scripts/build.sh`. This scr
 
 ### Run
 
-You can run you Precompile-EVM by using Avalanche Network Runner
+You can run you Precompile-EVM by using the Avalanche CLI.
+
+First, create the configuration for your subnet.
 
 ```bash
-avalanche-network-runner server \
---log-level debug \
---port=":8080" \
---grpc-gateway-port=":8081"
+avalanche subnet create mysubnet
+✔ Custom
+creating custom VM subnet mysubnet
+Enter path to custom genesis: ./.devcontainer/genesis-example.json
+✔ I already have a VM binary (local network deployments only)
+Enter path to VM binary: /go/src/github.com/ava-labs/avalanchego/build/plugins/srEXiWaHuhNyGwPUi444Tu47ZEDwxTWrbQiuD7FmgSAQ6X7Dy
```

Comment:
Can you check if we can use env variable`$AVALANCHEGO_PLUGIN_PATH` (/go/src/github.com/ava-labs/avalanchego/build/plugins/) here?